static/src/stylesheets/layout/new-header/_menu-item.scss

```diff
@@ -0,0 +1,127 @@
+.menu-item {
+    position: relative;
+    width: 100%;
+
+    > *[data-link-name='nav2 : the guardian app'],
+    > *[data-link-name='nav2 : facebook'] {
+        border-top: 1px solid darken($guardian-brand-dark, 4%);
+        margin-top: $gs-baseline / 1.5;
+        padding-top: $gs-baseline / 1.4;
+    }
+
+    > *[data-link-name='nav2 : the guardian app'] {
+        margin-top: $gs-baseline * 2;
+    }
+
+    // First level overwrites
+    .menu-group--primary & {
+        > *:not([open]) {
+            border-bottom: 1px solid darken($guardian-brand-dark, 4%);
+        }
+
+        &:last-child > * {
+            border-bottom: 0;
+        }
+    }
+}
+
+.menu-item__title {
+    color: currentColor;
+    cursor: pointer;
+    display: block;
+    font-size: 18px;
+    outline: none;
+    padding-bottom: $gs-baseline / 1.8;
```

Comment:
`$gs-baseline / 1.8` ?? Surely this should be just `$gs-baseline / 2`. I don't think we should promote things that go against our grid system that much